app/views/govuk_publishing_components/components/_copy_to_clipboard.html.erb

```diff
@@ -2,7 +2,7 @@
   button_data_attributes ||= nil
   input_data_attributes ||= nil
 %>
-<div class="gem-c-copy-to-clipboard" data-module="copy-to-clipboard">
+<div class="gem-c-copy-to-clipboard govuk-!-display-none-print" data-module="copy-to-clipboard">
```

Comment:
We chose to hide this for print as it's only useful in its interactive state on screen. I don't see the harm including it even if it's not used publicly: the change covers every scenario where it might be printed.

Reply:
Only problem is this component is a form field and we decided not to print form fields, and in this case it would be the only one in the form that won't appear in print, which might be a bit strange.

Reply:
I understand that point, but this component really is redundant in print so I'd prefer to remove it.

Reply:
I did mean that, sorry for the confusion. Thanks for updating.